nvi-evaluator/src/main/java/no/sikt/nva/nvi/evaluator/calculator/PointCalculator.java

```diff
@@ -126,6 +129,23 @@ private static BigDecimal getInstanceTypeAndLevelPoints(String instanceType,
         return INSTANCE_TYPE_AND_LEVEL_POINT_MAP.get(instanceType).get(channelType).get(level);
     }
 
+    private static boolean isInternationalCollaboration(JsonNode jsonNode) {
+        return getJsonNodeStream(jsonNode, JSON_PTR_CONTRIBUTOR)
+                   .filter(PointCalculator::isCreator)
+                   .flatMap(contributorNode -> getJsonNodeStream(contributorNode, JSON_PTR_AFFILIATIONS))
```

Comment:
Probably for both map ops. extract a named method to sy. what is being done here, so `.flatMap(this::extractAffiliation).map(this::extractCountryCode)` or a contraction of these